src/request_weaver.cc

```diff
@@ -229,7 +366,7 @@ void RequestWeaver::CollisionCheck(internal::string_view name) {
 
 RequestWeaver::WeaveInfo* RequestWeaver::WeaveInfo::FindWeaveMsg(
     const internal::string_view field_name) {
-  for (auto& msg : messages) {
+  for (auto& msg: messages) {
```

Comment:
against style guide?

Reply:
Reverted. It's auto changed by my clion.